.github/workflows/terraform-static-analysis.yml

```diff
@@ -42,5 +44,3 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           scan_type: full
```

Comment:
Shouldn't both test run configurations be using the same exclusions?

Reply:
> Shouldn't both test run configurations be using the same exclusions?

I am not sure why the same exclusions are not required during the second "full" scan, however, since the scan succeeds, I thought to remove them because they would have no impact. I think we should only add exclusions that are failing. Let me know if you still think otherwise.

Reply:
Exclusions re-added for manually triggered/full runs